docs/troubleshooting/kubernetes.rst

```diff
@@ -25,3 +235,226 @@ If you've seen this error, you can override or change the default traffic-group
 .. code-block:: javascript
 
    "trafficGroup": "/Common/traffic-group-local-only"
+
+-----------------------------------------
+
+.. _logging troubleshoot:
+
+Logging
+```````
+
+Set the log level
+~~~~~~~~~~~~~~~~~
+
+To change the log level for the |kctlr|:
+
+#. Edit the :ref:`Deployment <k8s-bigip-ctlr-deployment>` for each k8s-bigip-ctlr instance. Be sure to specify the correct namespace for the instance.
+
+   .. code-block:: bash
+
+      kubectl edit my-bigip-ctlr.yaml [--namespace kube-system]
+      oc edit my-bigip-ctlr.yaml [--namespace kube-system]
+
+#. Add the desired log-level to the :code:`args` section, then save and exit the editor.
+
+   For example: ::
+
+   --log-level=debug
+
+#. Verify the Deployment updated successfully.
+
+   .. code-block:: bash
+
+      kubectl describe deployment my-bigip-ctlr -o wide [--namespace kube-system]
+      oc describe deployment my-bigip-ctlr -o wide [--namespace kube-system]
+
+-----------------------------------------
+
+View the logs
+~~~~~~~~~~~~~
+
+- To view the logs:
+
+  .. code-block:: bash
+
+     kubectl logs my-bigip-ctlr-pod
```

Comment:
these both need the --namespace cube-system argument I believe. without it there are no logs returned on my system

Reply:
done